build-logic/settings/src/main/kotlin/com/squareup/anvil/builds/settings/SettingsPlugin.kt

```diff
@@ -40,6 +40,7 @@ abstract class SettingsPlugin @Inject constructor(
       repos.mavenCentral()
       repos.gradlePluginPortal()
       repos.google()
+      repos.mavenLocal()
```

Comment:
had to add this locally to be able to run the tests properly, otherwise it'd go to mavenCentral/google etc trying to find the annotations-jvm package but failed because it's new. I think this might have been an issue with the current impl?

Reply:
Mmm I guess we'll need to add a publishToMavenLocal to the CI bits that still fail 😭